keyboards/montsinger/rebound/rebound.h

```diff
@@ -26,7 +26,7 @@
  * represents the switch matrix.
  */
  
-#define LAYOUT_ortho_4x12( \
+#define LAYOUT_ortho4x12( \
```

Comment:
```suggestion
#define LAYOUT_ortho_4x12( \
```

Reply:
Ah, yes, my friend Underscore, glad to see y'all've met. Thanks for the help bud!